docs/conf/prague/2020/welcome-wagon.rst

```diff
@@ -1,194 +1,222 @@
-   exhausting. Give your brain a break! Grab a spot in the quiet room
-   or take a quick walk. Play a board game on your lunch break. Come
-   back invigorated.
--  Starting Monday morning, check the unconference schedule
-   to see if there are any sessions you are interested in
-   attending. New sessions are added all the time, so check back
-   periodically.
--  Eat! You can use the energy.
--  Are you looking for a job or is there an opening at your company?
-   Check out the job board in the unconference room.
-
-
-FAQ
----
-
-.. contents::
-   :local:
+- The Job Fair takes place on Tuesday morning. Each company attending the Job Fair will have their own Session table. To view the Job Fair participants and booths, go to **Sessions**.
+- Once you are in the session, check out the pinned message to review job and company information.
+- If you want to start a conversation with someone at the session, go to the **Chat** tab, make sure the **Session** toggle is on, and ask your question. You can set up 1:1 conversations using chat or video, too.
+- Remember, there might be multiple people starting conversations, so there could be a wait.
+
+Sponsor booths
+--------------
+
+Select **Expo** to view Sponsor booths. Sponsors might have videos you can watch, information to review on the **Booth** tab, and people you can talk to on the **Chat** tab. Visiting Sponsor booths is a great way to make connections!
+
+Monday Social
+-------------
+
+The Monday night Social will be designed by you this year! You can host a session and meet fellow attendees for a video social event or you can attend a social event set up by someone else.
+
+Attend a social event
+~~~~~~~~~~~~~~~~~~~~~
+
+Check the `list of social events <https://docs.google.com/spreadsheets/d/1HPAXim5gkPmwhANJwngWcg1d09KHqOj3vILn5Tc5V9A/edit#gid=1493675134>`__\ **,** and then go to the **Sessions** page and select the table that you want to join.
+
+Host a social event
+~~~~~~~~~~~~~~~~~~~
+
+If you have an idea for a social event, find an empty table in the `sign-up sheet <https://docs.google.com/spreadsheets/d/1HPAXim5gkPmwhANJwngWcg1d09KHqOj3vILn5Tc5V9A/edit#gid=1493675134>`__ and add your session. Here’s some ideas:
+
+- Online game night
+- WFH life meetup – give attendees a video tour of your space
+- Music performance
+- YouTube karaoke
+- Silly hats/furry friends meetup
+
+Networking
+----------
+
+Add info to your profile
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To add information about yourself to your conference profile, select your initials > **Edit Profile**. Share as much as you are comfortable.
+
+Important: Any info you add to your profile can be seen by everyone attending the conference. All conversations should adhere to the `Write the Docs Code of Conduct <https://www.writethedocs.org/code-of-conduct/>`__.
+
+Reach out to attendees
+~~~~~~~~~~~~~~~~~~~~~~
+
+There are several ways to network using the virtual conference platform:
+
+Networking
```

Comment:
This section needs to be removed because we're not using that component anymore

Reply:
I replaced this with the hallway sessions.